Satellite Charts

urchykli opened the pull request:
The three charts are complete, but I wasn't sure about the dates for the Long March Launches chart. Should I show all years on the x axis like in the google sheet or leave it as is? Thanks!

urchykli commented:
All four charts are now included. Thanks!

jnschrag commented on satellites/launches/js/bar.js:
```diff
@@ -0,0 +1,71 @@
+  },
+  // Tooltip
+  tooltip: {
+    valueSuffix: " Cinema Screens",
```
This should probably be removed :)

jnschrag commented on satellites/payloads/js/lines.js:
```diff
@@ -0,0 +1,63 @@
+  },
+  // Tooltip
+  tooltip: {
+    valueSuffix: " Cinema Screens",
```
This should be removed.

jnschrag commented on satellites/payloads/js/lines.js:
```diff
@@ -0,0 +1,63 @@
+    type: "line"
+  },
+  // Colors
+  colors: ['#ed392a'],
```
No need to explicitly define colors here. It'll pull them automatically in order, so red would be the first one called.

urchykli commented:
I've made the following updates.
- Shared tooltip for Launches
- Removed Cinema Screens from all charts
- Link to remote Highcharts theme
- Remove unnecessary color from Payloads
- Removed "year" from all charts
- Update color pallet for pie chart
- Rename directory to Space Capabilities

Thanks!

urchykli commented:
Per request from the program, I've made the following changes.
- Updated theme on china-space-timeline/humans
- Updated tooltip on Launches chart
- Changed purple to green on the Launches chart
- Made "per" lowercase on the Payloads chart

Thanks!
